src/components/admin/article/ArticleController.jsx

```diff
@@ -385,67 +454,82 @@ class ArticleController extends FalcorController {
 
       const { id } = this.props.params;
       const article = this.state.data.articles.byId[id];
-
       // If it is a new article it won't have any meta data yet so we use the default
       const categories = _.toArray(this.state.data.categories.byIndex);
       categories.push({ name: 'none', id: null });
-      const actionButtons = [
-        <SaveButton
-          onClick={this.handleSaveChanges}
-          style={styles.buttons}
-          saving={this.state.saving}
-          changed={this.state.changed}
-        />,
-      ];
 
       const articleEditorID = 'article-editor';
       return (
-        <div id={articleEditorID}>
-          <Dialog
-            title="Article Editor"
-            actions={actionButtons}
-            open
-            modal={false}
-            autoScrollBodyContent
-            onRequestClose={this.handleDialogClose}
-          >
-            {this.state.saving ? <LoadingOverlay /> : null}
+        <Paper id={articleEditorID} style={styles.grid}>
+          {this.state.saving ? <LoadingOverlay /> : null}
+
+          <div style={styles.header}>
+            <h2 style={{ display: 'inline' }}>Article Editor</h2>
+            <span
+              style={{
+                color: 'rgba(0, 0, 0, 0.5)',
+                fontSize: '0.8vw',
+                marginLeft: '1vw',
+              }}
+            >
+              {this.props.published_at !== null
```

Comment:
that prop doesn't exist. It should be `article.published_at`, right now it's outputting undefined which is `!== null` which moment apparently takes as it should output current date, so it always outputs the current date